commit bf51ffad4b64d5b04d76715fc53f5174ece07af2
Author: Michael Zimmermann <sigmaepsilon92@gmail.com>
Date:   Thu Mar 26 18:16:08 2026 +0100

    BUILDROOT

diff --git a/cmake/dependencies/common.cmake b/cmake/dependencies/common.cmake
index 97319be6..1b71abb3 100644
--- a/cmake/dependencies/common.cmake
+++ b/cmake/dependencies/common.cmake
@@ -24,7 +24,7 @@ pkg_check_modules(CURL REQUIRED libcurl)
 
 # miniupnp
 pkg_check_modules(MINIUPNP miniupnpc REQUIRED)
-include_directories(SYSTEM ${MINIUPNP_INCLUDE_DIRS})
+include_directories(${MINIUPNP_INCLUDE_DIRS})
 
 # ffmpeg pre-compiled binaries
 if(NOT DEFINED FFMPEG_PREPARED_BINARIES)
diff --git a/cmake/dependencies/libevdev_Sunshine.cmake b/cmake/dependencies/libevdev_Sunshine.cmake
index ca9e22dc..0e374b6a 100644
--- a/cmake/dependencies/libevdev_Sunshine.cmake
+++ b/cmake/dependencies/libevdev_Sunshine.cmake
@@ -40,7 +40,7 @@ if(EVDEV_INCLUDE_DIR AND EVDEV_LIBRARY)
     message(STATUS "Found libevdev library: ${EVDEV_LIBRARY}")
     message(STATUS "Found libevdev include directory: ${EVDEV_INCLUDE_DIR}")
 
-    include_directories(SYSTEM ${EVDEV_INCLUDE_DIR})
+    include_directories(${EVDEV_INCLUDE_DIR})
     list(APPEND PLATFORM_LIBRARIES ${EVDEV_LIBRARY})
 else()
     message(FATAL_ERROR "Couldn't find or fetch libevdev")
